api/mapistore_http/source/index.rst

```diff
@@ -436,24 +431,70 @@ Calendar
       Content-Type: application/json
 
       {
-        "id": "15924213158245d0ad631c6a41a0e7c3"
+        "id": "51c3187152d0a0daa5e0de4d6e3132cb561135e7"
       }
 
-      :>json string id: Message identifier of the calendar item created
+      :>json string id: Message identifier of the mail item created
       :reqheader Authorization: auth token
       :statuscode 200: Ok
 
 
-.. http:get:: /calendars/(id)/
+.. http:post:: /mails/submit/
 
-   :synopsis: Retrieve all the properties of the calendar entry
+   :synopsis: Send an E-mail message to its recipients.
```

Comment:
If we use submit as verb in REST, we should use submit in the synopsis, shouldn't we?